edx_sga/static/js/src/edx_sga.js

```diff
@@ -330,6 +331,8 @@ function StaffGradedAssignmentXBlock(runtime, element) {
                 block.find('#grade-submissions-button')
                     .leanModal()
                     .on('click', function() {
+                        sendResizeMessage(document.body.clientHeight * 2)
```

Comment:
I have tried it but when I keep the page content that spans to two pages. So, when I open up that popup it spans to four pages that doesn't make sense at all

I have attached a picture for your reference
[screencapture-localhost-2000-course-course-v1-edX-DemoX-Demo-Course-block-v1-edX-DemoX-Demo-Course-type-sequential-block-basic-questions-block-v1-edX-DemoX-Demo-Course-type-vertical-block-2152d4a4aadc4cb0af5256394a3d1fc7-2021-12-28-18_40_57.pdf](https://github.com/mitodl/edx-sga/files/7784930/screencapture-localhost-2000-course-course-v1-edX-DemoX-Demo-Course-block-v1-edX-DemoX-Demo-Course-type-sequential-block-basic-questions-block-v1-edX-DemoX-Demo-Course-type-vertical-block-2152d4a4aadc4cb0af5256394a3d1fc7-2021-12-28-18_40_57.pdf)

[screencapture-localhost-2000-course-course-v1-edX-DemoX-Demo-Course-block-v1-edX-DemoX-Demo-Course-type-sequential-block-basic-questions-block-v1-edX-DemoX-Demo-Course-type-vertical-block-2152d4a4aadc4cb0af5256394a3d1fc7-2021-12-28-18_41_38.pdf](https://github.com/mitodl/edx-sga/files/7784932/screencapture-localhost-2000-course-course-v1-edX-DemoX-Demo-Course-block-v1-edX-DemoX-Demo-Course-type-sequential-block-basic-questions-block-v1-edX-DemoX-Demo-Course-type-vertical-block-2152d4a4aadc4cb0af5256394a3d1fc7-2021-12-28-18_41_38.pdf)

Reply:
On small pages it looks like that is also very weird

<img width="1440" alt="Screenshot 2021-12-28 at 18 10 58" src="https://user-images.githubusercontent.com/4043989/147572650-7a8cf58c-55c5-4f27-9697-d73bdc1bf620.png">
<img width="1436" alt="Screenshot 2021-12-28 at 18 44 54" src="https://user-images.githubusercontent.com/4043989/147572663-e6d23fa1-3835-4158-b6f3-fefc4d8cb4b9.png">
<img width="1440" alt="Screenshot 2021-12-28 at 18 45 01" src="https://user-images.githubusercontent.com/4043989/147572667-a32e7def-f956-41f0-8e5e-ed38ecc8338b.png">
<img width="1440" alt="Screenshot 2021-12-28 at 18 45 09" src="https://user-images.githubusercontent.com/4043989/147572670-bf8fec0a-2ac6-432d-af30-e1c614b5be05.png">